Assets/MixedRealityToolkit.Tests/EditModeTests/InputSystem/InputSystemTests.cs

```diff
@@ -12,8 +12,13 @@ namespace Microsoft.MixedReality.Toolkit.Tests.EditMode.InputSystem
 {
     public class InputSystemTests
     {
-        private const string TestInputSystemProfilePath = "Assets/MixedRealityToolkit.Tests/EditModeTests/Services/TestProfiles/TestMixedRealityInputSystemProfile.asset";
-        private const string TestEmptyInputSystemProfilePath = "Assets/MixedRealityToolkit.Tests/EditModeTests/Services/TestProfiles/TestEmptyMixedRealityInputSystemProfile.asset";
+        // Tests/EditModeTests/Services/TestProfiles/TestMixedRealityInputSystemProfile.asset
+        private const string TestInputSystemProfileGuid = "9c5be3fea81382342b84623b5b225012";
+        private static readonly string TestInputSystemProfilePath = AssetDatabase.GUIDToAssetPath(TestInputSystemProfileGuid);
+
+        // Tests/EditModeTests/Services/TestProfiles/TestEmptyMixedRealityInputSystemProfile.asset
+        private const string TestEmptyInputSystemProfileGuid = "45c96c8b3073919419eae31f72ca8e4e";
```

Comment:
Would it be easier to use the MRTK files lookup? This feels like adding steps (open meta file, find guid, etc) for developers.

Reply:
on the other hand, is this a faster access method for test collateral (ex: no file system lookup)

Reply:
My thought was that the MRTKFiles APIs seemed to do a lot regarding searching directories and everything. Depending on how Unity maps GUIDs to paths, this felt like a more performant solution, and one that would be more robust to file name or folder name changes.

I can run some tests today to see perf between the two.

Reply:
Ran the following test. Let me know if it looks like I misconfigured anything, but the results were 

![image](https://user-images.githubusercontent.com/3580640/75574959-17997480-5a14-11ea-89a6-a55f44eb5d4b.png)


```c#
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

using Microsoft.MixedReality.Toolkit.Input;
using Microsoft.MixedReality.Toolkit.SpatialAwareness;
using Microsoft.MixedReality.Toolkit.Utilities.Editor;
using System.Diagnostics;
using UnityEditor;
using UnityEngine;

public class TestFileLoading : MonoBehaviour
{
    private const string TestInputSystemProfileGuid = "9c5be3fea81382342b84623b5b225012";
    private const string TestEmptyInputSystemProfileGuid = "45c96c8b3073919419eae31f72ca8e4e";
    private const string TestSpatialAwarenessSystemProfileGuid = "757d72d70f9c56144b05346288682e00";
    private const string CameraSystemPrefabGuid = "8a5d96f565aaee14ca3e07575bb9ce80";
    private const string RadialSetPrefabAssetGuid = "8b83134143223104c9bc3865a565cab3";

    private const string TestInputSystemProfilePath = "EditModeTests/Services/TestProfiles/TestMixedRealityInputSystemProfile.asset";
    private const string TestEmptyInputSystemProfilePath = "EditModeTests/Services/TestProfiles/TestEmptyMixedRealityInputSystemProfile.asset";
    private const string TestSpatialAwarenessSystemProfilePath = "EditModeTests/Services/TestProfiles/TestMixedRealitySpatialAwarenessSystemProfile.asset";
    private const string CameraSystemPrefabPath = "Experimental/ServiceManagers/Camera/Prefabs/CameraSystem.prefab";
    private const string RadialSetPrefabAssetPath = "Features/UX/Interactable/Prefabs/RadialSet.prefab";

    private void Start()
    {
        Stopwatch stopwatch = Stopwatch.StartNew();

        for (int i = 0; i < 1000; i++)
        {
            _ = AssetDatabase.LoadAssetAtPath<MixedRealityInputSystemProfile>(AssetDatabase.GUIDToAssetPath(TestInputSystemProfileGuid));
            _ = AssetDatabase.LoadAssetAtPath<MixedRealityInputSystemProfile>(AssetDatabase.GUIDToAssetPath(TestEmptyInputSystemProfileGuid));
            _ = AssetDatabase.LoadAssetAtPath<MixedRealitySpatialAwarenessSystemProfile>(AssetDatabase.GUIDToAssetPath(TestSpatialAwarenessSystemProfileGuid));
            _ = AssetDatabase.LoadAssetAtPath<Object>(AssetDatabase.GUIDToAssetPath(CameraSystemPrefabGuid));
            _ = AssetDatabase.LoadAssetAtPath<Object>(AssetDatabase.GUIDToAssetPath(RadialSetPrefabAssetGuid));
        }

        stopwatch.Stop();

        UnityEngine.Debug.Log($"Load by GUID took {stopwatch.ElapsedMilliseconds} milliseconds.");

        stopwatch.Reset();
        stopwatch.Start();

        for (int i = 0; i < 1000; i++)
        {
            _ = AssetDatabase.LoadAssetAtPath<MixedRealityInputSystemProfile>(MixedRealityToolkitFiles.MapRelativeFilePath(MixedRealityToolkitModuleType.Tests, TestInputSystemProfilePath));
            _ = AssetDatabase.LoadAssetAtPath<MixedRealityInputSystemProfile>(MixedRealityToolkitFiles.MapRelativeFilePath(MixedRealityToolkitModuleType.Tests, TestEmptyInputSystemProfilePath));
            _ = AssetDatabase.LoadAssetAtPath<MixedRealitySpatialAwarenessSystemProfile>(MixedRealityToolkitFiles.MapRelativeFilePath(MixedRealityToolkitModuleType.Tests, TestSpatialAwarenessSystemProfilePath));
            _ = AssetDatabase.LoadAssetAtPath<Object>(MixedRealityToolkitFiles.MapRelativeFilePath(MixedRealityToolkitModuleType.SDK, CameraSystemPrefabPath));
            _ = AssetDatabase.LoadAssetAtPath<Object>(MixedRealityToolkitFiles.MapRelativeFilePath(MixedRealityToolkitModuleType.SDK, RadialSetPrefabAssetPath));
        }

        stopwatch.Stop();

        UnityEngine.Debug.Log($"Load by MRTKFiles took {stopwatch.ElapsedMilliseconds} milliseconds.");
    }
}
```

Reply:
It is negligible in our scenario, but very enlightening. I'm good with using GUIDs provided that we always use them to access test collateral :)